examples/cpp/modes/goto/include/util.hpp

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+#pragma once
+
+#include <Eigen/Eigen>
+#include <cmath>
+
+static inline Eigen::Vector3f quaternionToEuler(const Eigen::Quaternionf & q)
```

Comment:
These should be made generally available under some `math/conversion` dir

Reply:
yeah i agree .. just didnt do it yet for the sake of getting the example out quick - added ugly note to shame myself until we consider where to better move these utils